src/wpt/BUILD.bazel

```diff
@@ -2,21 +2,29 @@ load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
 load("@npm//:eslint/package_json.bzl", eslint_bin = "bin")
 load("//:build/wpt_test.bzl", "wpt_test")
 
-harness_srcs = glob(
-    [
-        "harness/*.ts",
-    ],
+wpt_test(
+    name = "url",
+    config = "url-test.ts",
+    wpt_directory = "@wpt//:url",
 )
 
-test_srcs = glob(["**/*-test.ts"])
+wpt_test(
+    name = "urlpattern-old",
+    config = "urlpattern-old-test.ts",
+    wpt_directory = "@wpt//:urlpattern",
+)
```

Comment:
@anonrig You could add something like this for your test:

```
wpt_test(
    name = "urlpattern-ada",
    config = "urlpattern-ada-test.ts",
    wpt_directory = "@wpt//:urlpattern",
    autogates = ["workerd-autogate-urlpattern"]
)
```

Reply:
Perfect. Can you remove the renaming of the urlpattern test? I'll change the new test file to urlpattern-standard to follow the same pattern in url

Reply:
Done